tests/src/CMakeLists.txt

```diff
@@ -88,7 +96,9 @@ target_compile_features(testfunctiontraits PUBLIC cxx_std_20)
 
 set(TEST_DEPENDING_ON_IGA testklshell testadaptivestepsizing)
 
-set(TEST_NEED_MORE_TIME testadaptivestepsizing)
+set(TEST_NEED_MORE_TIME testnonlineareas testadaptivestepsizing testcantileverbeamAS
+                        testcantileverbeamEAS
```

Comment:
I think in Ikarus, `testcantileverbeamAS` and `testcantileverbeamEAS` need not be here as it works automatically nicely somehow. That's why I also removed `testnonlineareas` from here previously and can also be removed now